.travis.yml

```
@@ -30,7 +30,7 @@ script:
 
 # End to End test
   - docker run --name todo-app-e2eTest
-    -e SECRET_APIKEY=$SECRET_APIKEY -e SECRET_APITOKEN=$SECRET_APITOKEN -e SECRET_KEY=secret-key -e BOARD_NAME=Sqmiler-Dev
+    -e USER_NAME=$USER_NAME -e PASSWORD=$PASSWORD -e MONGO_URI=$MONGO_URI -e MONGO_PROT=$MONGO_PROT -e DEFAULT_DATABASE=test_board_name -e SECRET_KEY=secret-key
```

Really minor, but a slight shorthand available where the variable names match inside/outside the container
```suggestion
    -e USER_NAME -e PASSWORD -e MONGO_URI -e MONGO_PROT -e DEFAULT_DATABASE=test_board_name -e SECRET_KEY=secret-key
```